src/Amaro.js

```diff
@@ -21,14 +21,14 @@ const shaders = GL.Shaders.create({
         vec4 texel = texture2D(inputImageTexture, uv);
         vec3 bbTexel = texture2D(inputImageTexture2, uv).rgb;
 
-        texel.r = texture2D(inputImageTexture3, vec2(bbTexel.r, texel.r)).r;
-        texel.g = texture2D(inputImageTexture3, vec2(bbTexel.g, texel.g)).g;
-        texel.b = texture2D(inputImageTexture3, vec2(bbTexel.b, texel.b)).b;
+        texel.r = texture2D(inputImageTexture3, vec2(bbTexel.r, (1.0-texel.r))).r;
```

Comment:
Ok, so we need to flp the y axis. Nice pick up. Any idea why this is the case? Is it something that react gl does?